M2/Macaulay2/editors/Macaulay2Web/M2-symbols.ts.in

```diff
@@ -1 +1 @@
-export default [@M2SYMBOLS@];
+export default [@M2SYMBOLS@].sort();
```

Comment:
Does sorting this list matter? If it does, shouldn't it be sorted in `make-M2-symbols`?

Reply:
the problem is, M2 uses nonstandard ordering for characters:
```
i1 : sort{"a","b","A","B"}

o1 = {A, a, B, b}

o1 : List
```

Reply:
Why does it need to be sorted?

Reply:
I think it's intentional, though I don't remember the reason now. It is a pain though.
It needs to be sorted because it's used for autocomplete which goes through the list to find matches.

Reply:
Okay, then please change the order in `make-M2-symbols` this way:
```
i15 : exportFrom_Core "sortBy";

i16 : (sortBy ascii) {"a","b","A","B"}

o16 = {A, B, a, b}
```

Reply:
OK. If the other generated files don't care what the ordering is, then it is the best fix, yes. done.